src/controllers/job/Transactions.ts

```diff
@@ -118,7 +120,13 @@ const formatDate = ({ date, format }: IformatDate) => {
 
 export default {
   crawl: async (req: Request, res: Response) => {
-    const allStock: Stocks[] = await Stocks.findAll();
+    const stock: Stocks = await Stocks.findOne({ where: req.body });
```

Comment:
should be define variable
const code = req.body.stock_code || '';
const stock: Stocks = await Stocks.findOne({ where: { code } });